lib/situation_room_web/router.ex

```diff
@@ -6,7 +6,7 @@ defmodule SituationRoomWeb.Router do
     plug(:fetch_session)
     plug(:fetch_live_flash)
     plug(:put_root_layout, {SituationRoomWeb.LayoutView, :root})
-    plug(:protect_from_forgery)
+    #plug(:protect_from_forgery)
```

Comment:
What is the story here?  Can this either be removed or added back in?

Reply:
Probably shouldn't have been committed but it needs to be there eventually, we just couldn't post from postman etc to add sites without getting a csrf token to pass. Maybe should be optionally included for prod